internal/pkg/service/buffer/sink/tablesink/storage/repository/hook.go

```diff
@@ -66,10 +66,10 @@ func (h *hook) AssignVolumes(_ context.Context, allVolumes []volume.Metadata, cf
 }
 
 func (h *hook) NewFileResourcesProvider(rb rollback.Builder) FileResourcesProvider {
-	result := make(map[key.SinkKey]FileResource)
+	result := make(map[key.SinkKey]*FileResource)
```

Comment:
Modified map type, used pointer:
- Reading from the map is now easier, `nil' means that the value is not there.

Reply:
https://github.com/keboola/keboola-as-code/pull/1522#discussion_r1457205223